src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -358,9 +362,70 @@ private static void ConfigureAuthentication(WebApplicationBuilder builder, Dashb
                 };
             });
 
+        if (dashboardStartupConfig.WebAppAuthMode == WebAppAuthMode.OpenIdConnect)
```

Comment:
Is there a downside to always registering these?

Reply:
Loads a bunch of code into the process that most users will never need. If there's an upside I'm missing, let me know. We're unlikely to ever make the auth settings changeable throughout the process lifetime.